classes/class3/cat-app/views/color.handlebars

```diff
@@ -0,0 +1,7 @@
+<h1>Color</h1>
+
+<ul>
+{{#each cats}}
+	{{this.name}}
```

Comment:
You've created a `<ul>` list here but aren't placing the contents in individual `<li>` tags.